lib/ardb/has_slug.rb

```diff
@@ -53,11 +53,12 @@ def reset_slug
 
       def ardb_has_slug_generate_slug
         slug_attr_value = self.send(self.class.ardb_has_slug_config[:attribute])
```

Comment:
@jcredding if you don't like `a` as the local var name, maybe `attr_name` would work better?